AllReadyApp/Web-App/AllReady/Features/Notifications/NotifyAssignedVolunteersForTheTask.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System.Linq;
+using System.Threading.Tasks;
+using AllReady.Models;
+using MediatR;
+
+namespace AllReady.Features.Notifications
+{
+    public class NotifyAssignedVolunteersForTheTask : IAsyncNotificationHandler<TaskAssignedToVolunteersNotification>
+    {
+        private readonly AllReadyContext _context;
+        private readonly IMediator _mediator;
+
+        public NotifyAssignedVolunteersForTheTask(AllReadyContext context, IMediator mediator)
+        {
+            _context = context;
+            _mediator = mediator;
+        }
+
+        public async Task Handle(TaskAssignedToVolunteersNotification notification)
+        {
+            var users = _context.Users.Where(x => notification.NewlyAssignedVolunteers.Contains(x.Id)).ToList();
```

Comment:
@stevejgordon Good catch. will do. I don't know the business reasoning on unassigning users.